app/resources/scanned_resources/letter_change_set.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+class LetterChangeSet < ChangeSet
+  apply_workflow(DraftCompleteWorkflow)
+  core_resource(change_set: "letter")
+  enable_order_manager
+  enable_pdf_support
+
+  property :sender, multiple: false, required: false, form: NameWithPlaceChangeSet, populator: :populate_name_with_place
+  property :recipient, multiple: false, required: false, form: NameWithPlaceChangeSet, populator: :populate_name_with_place
+
+  def primary_terms
+    feature_terms.dup.insert(
```

Comment:
this is a little awkward, but in general i like the property shortcuts idea.  A bit of documentation to go along with it in the figgy wiki? If so, pointing out this pattern is probably a good idea.

Reply:
Added to https://github.com/pulibrary/figgy/wiki/ChangeSets-in-Figgy